tests/unit_tests/data_model/test_format_checkers.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+
+import pytest
+
+from simtools.data_model import format_checkers
+
+
+def test_check_astropy_unit():
+    assert format_checkers.check_astropy_unit("dimensionless")
+    assert format_checkers.check_astropy_unit("m")
+
+
+def test_check_array_elements():
+    assert format_checkers.check_array_element("MSTN-15")
+
+    with pytest.raises(ValueError, match=r"^Invalid name"):
```

Comment:
Oh, I see that indeed it raises a `ValueError`. So when would it raise `ValidationError`? Or is this test not complete?

Reply:
Never. Why do you expect a ValidationError ? And this would be a `jsonschema.exceptions.ValidationError`

Reply:
Cause I mixed up this and the other function which I thought it calls `_validate_data_dict_using_json_schema`. I pulled out the branch and looked at the code more carefully now and this comment is simply wrong.